packages/content/src/hub.ts

```diff
@@ -6,6 +6,12 @@ import {
   hubApiRequest
 } from "@esri/hub-common";
 import { itemToContent, withPortalUrls } from "./portal";
+import { isSlug, addContextToSlug } from "./slugs";
+import { cloneObject } from "@esri/hub-common";
+
+export interface IGetContentOptions extends IHubRequestOptions {
+  siteOrgKey?: string;
```

Comment:
@dbouwman I contemplated putting something like `site` on `IHubRequestOptions` as a place for the global site/domain data we store in appSettings in the Ember app b/c I suspect we'll need that stuff in a few places, but ended up just stabbing the one prop I need onto the local options for this fn.

Reply:
I'd be ok adding it as an optional on `IHubRequestOptions` but we def can't make it required as there are scenarios where this is no "site" context... and then there's the issue that sometimes the siteId is enough, others you'd really need the whole model, and in some env's you'd have it (i.e. Ember), others (i.e. scripting in node, or in some rando component) you may not... it's all messy messy with few clear "winner" solutions

Reply:
yeah, that's sort of where I got to, and why I decided to just punt and add only exactly what I needed for this fn.